File: solutions/uri/1893/1893.cpp
#include <iostream>

int main() {
    int16_t first_night, second_night;

    while (std::cin >> first_night >> second_night) {
        if (second_night <= 2) {
            std::cout << "nova" << std::endl;
        } else if (second_night >= 97) {
            std::cout << "cheia" << std::endl;
        } else if (first_night >= second_night) {
            std::cout << "minguante" << std::endl;
        } else {
            std::cout << "crescente" << std::endl;
        }
    }

    return 0;
}

File: solutions/beecrowd/1893/1893.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

while line := input().strip():
    first_night, second_night = map(int, line.split())
    if second_night <= 2:
        output('nova\n')
    elif second_night >= 97:
        output('cheia\n')
    elif first_night >= second_night:
        output('minguante\n')
    else:
        output('crescente\n')
